packages/web-components/src/card/index.ts

```diff
@@ -1,77 +1,103 @@
+import { composedParent, Card as FoundationCard, CardTemplate as template } from '@microsoft/fast-foundation';
 import { attr, Notifier, Observable } from '@microsoft/fast-element';
 import { parseColorHexRGB } from '@microsoft/fast-colors';
-import { designSystemProvider, CardTemplate as template } from '@microsoft/fast-foundation';
-import { neutralFillCard } from '../color';
-import { DesignSystem } from '../fluent-design-system';
-import { FluentDesignSystemProvider } from '../design-system-provider';
+import { fillColor, neutralFillLayerRecipe, neutralPalette } from '../design-tokens';
+import { Swatch, SwatchRGB } from '../color-vNext/swatch';
+import { PaletteRGB } from '../color-vNext/palette';
 import { CardStyles as styles } from './card.styles';
 
 /**
- * The Fluent Card Element. Implements {@link @microsoft/fast-foundation#Card},
- * {@link @microsoft/fast-foundation#CardTemplate}
- *
- *
  * @public
- * @remarks
- * HTML Element: \<fluent-card\>
  */
-@designSystemProvider({
-  name: 'fluent-card',
-  template,
-  styles,
-  shadowOptions: {
-    mode: 'closed',
-  },
-})
-export class FluentCard extends FluentDesignSystemProvider {
+export class Card extends FoundationCard {
   /**
-   * Background color for the card component. Sets context for the design system.
+   * Fill color for the card component. Sets context for the design system.
    * @public
    * @remarks
-   * HTML Attribute: card-background-color
+   * HTML Attribute: card-fill-color
    */
   @attr({
-    attribute: 'card-background-color',
+    attribute: 'card-fill-color',
     mode: 'fromView',
   })
-  public cardBackgroundColor: string;
-  private cardBackgroundColorChanged(prev: string | void, next: string | void): void {
+  public cardFillColor: string;
+  private cardFillColorChanged(prev: string | void, next: string | void): void {
     if (next) {
-      const parsedColor = parseColorHexRGB(this.cardBackgroundColor);
+      const parsedColor = parseColorHexRGB(next);
 
       if (parsedColor !== null) {
-        this.neutralBaseColor = this.cardBackgroundColor;
-        this.backgroundColor = this.cardBackgroundColor;
+        this.neutralBaseColor = next;
+        fillColor.setValueFor(
+          this,
+          SwatchRGB.create(parsedColor.r, parsedColor.g, parsedColor.b)
+        );
       }
-    } else if (this.provider && this.provider.designSystem) {
-      this.handleChange(this.provider.designSystem as DesignSystem, 'backgroundColor');
+    }
+  }
+
+  /**
+   * Neutral palette base color for the card component. Sets context for the design system.
+   * @public
+   * @remarks
+   * HTML Attribute: neutral-base-color
+   */
+  @attr({
+    attribute: 'neutral-base-color',
+    mode: 'fromView',
+  })
+  public neutralBaseColor: string;
+  private neutralBaseColorChanged(prev: string | void, next: string | void): void {
+    if (next) {
+      const color = parseColorHexRGB(next)!;
```

Comment:
I'm curious if this is still needed. In general, shouldn't setting the card fill color just generate and apply a new neutral palette?

Reply:
There is probably a cleaner way to separate this, but this supports two use cases:
- Setting the card to a fixed solid color, like a bold, exact blue color
- Tinting the card but falling into the light and dark mode

This supports the second option, where you don't want the card to be exactly this blue, but you want to tint the neutral palette and stay in light or dark from the container.

Reply:
Do you think it would be cleaner to to change this to `neutralPalette` and accept a palette then? This implementation strikes me as designed around the old implementation where palettes and base-colors weren't associated directly.

Reply:
I do feel like this is a holdover and we're keeping it for compatibility for now. That might be a better way to do it, but I would expect something called `neutralPalette` to take the entire palette object. Maybe there's a better name for something constructed from something else like this. Like `neutralPaletteSource`. At least that ties the two together better.